x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -422,13 +430,95 @@ export class IndexActionsContextMenu extends Component {
                   }
                 )
               }
-              color="warning"
+              color={Boolean(kbnIndex) ? 'danger' : 'warning'}
+              iconType="help"
+            >
+              <p>
+                {Boolean(kbnIndex) ? (
+                  <FormattedMessage
+                    id="xpack.idxMgmt.indexActionsMenu.deleteIndex.deleteDangerDescription"
+                    defaultMessage="You are about to delete {kbnIndexName} index.
```

Comment:
I was thinking we could create a whole new modal that's designed to make the user pay more attention, something like this:

![image](https://user-images.githubusercontent.com/1238659/59621455-30f7cc00-90e4-11e9-8c91-752cf3e98452.png)

In the event that the user has selected non-system-indices as well, we could add a second list below the list of system indices:

```
You are also about to delete these regular indices:

  * index1
  * index2
```

Reply:
Instead of "Proceed with caution!" I would even put something like "Make sure your data is backed up" as the callout title. This way the modal looks different so if people scan it they'll hopefully notice that they're doing something unusual.